TESTS/netsocket/README.md

```diff
@@ -102,35 +102,41 @@ time        stream  tcp6    nowait  root    internal
 You can connect to the test server with an NMAP tool like this:
 
 ```.sh
-$ nmap -sT -p7,9,13,37 echo.mbedcloudtesting.com
+$ nmap -sT -p7,9,13,37,2007,2009,2013 echo.mbedcloudtesting.com
 
-Starting Nmap 7.12 ( https://nmap.org ) at 2018-04-05 16:17 EEST
+Starting Nmap 7.01 ( https://nmap.org ) at 2019-01-07 16:45 CET
 Nmap scan report for echo.mbedcloudtesting.com (52.215.34.155)
-Host is up (0.065s latency).
-Other addresses for echo.mbedcloudtesting.com (not scanned): 2a05:d018:21f:3800:8584:60f8:bc9f:e614
+Host is up (0.027s latency).
+Other addresses for echo.mbedcloudtesting.com (not scanned): 2a05:d018:21f:3800:3164:2a5c:75b3:970b
 rDNS record for 52.215.34.155: ec2-52-215-34-155.eu-west-1.compute.amazonaws.com
-PORT STATE SERVICE
-7/tcp open echo
-9/tcp open discard
-13/tcp open daytime
-37/tcp open time
+PORT     STATE SERVICE
+7/tcp    open  echo
+9/tcp    open  discard
+13/tcp   open  daytime
+37/tcp   open  time
+2007/tcp open  dectalk
+2009/tcp open  news
+2013/tcp open  raid-am
 
-Nmap done: 1 IP address (1 host up) scanned in 0.17 seconds
+Nmap done: 1 IP address (1 host up) scanned in 0.10 seconds
 
-$ sudo nmap -sT -p7,9,13,37 echo.mbedcloudtesting.com
+$ sudo nmap -sT -p7,9,13,37,2007,2009,2013 echo.mbedcloudtesting.com
```

Comment:
I can see that there was a error on previous documentation.
This was supposed to do UDP scan, and therefore required `sudo`

Can you change it to do
`sudo nmap -sU` instead?